client/settings/settings-manager/style.scss

```
@@ -0,0 +1,52 @@
+body {
+	/* this can be removed after all the settings sections are converted to the new layout */
+	&.woocommerce_page_wc-settings {
+		&.woocommerce_page_wc-settings-checkout {
+			background: #fff;
+			.settings-content {
```

```suggestion
			.settings-content,
			#wpbody-content,
			#wpcontent {
				background: #fff;
			}
```
We can combine the selectors using commas to avoid repeating the background color declaration.

---

changed in [ee1a5c6](https://github.com/Automattic/woocommerce-payments/pull/10578/commits/ee1a5c667ebf360eac3c476d55870eacc1c883e3). thx